letsencrypt/defaults.yaml

```diff
@@ -2,25 +2,30 @@
 # vim: ft=yaml
 ---
 letsencrypt:
-  use_package: true
+  install_method: package
```

Comment:
This change from `use_package` to `install_method` is the potential breaking change that I mentioned above.

Reply:
As far as I can see, the PR is backward compatible, as @litnialex suggests.